soccer/gameplay/tactics/one_touch_pass.py

```diff
@@ -0,0 +1,72 @@
+import composite_behavior
+import behavior
+import skills.move
+import tactics.coordinated_pass
+import robocup
+import constants
+import main
+import skills.angle_receive
+import evaluation.touchpass_positioning
+from enum import Enum
+
+## A tactic that causes a robot to pass to another one,
+# who scores on the goal as fast as possible.
+#
+# This class is supplemented by touchpass_positioning and angle_receive
+class OneTouchPass(composite_behavior.CompositeBehavior):
+
+    tpass = evaluation.touchpass_positioning.TouchpassPositioner()
+    THRESHOLD = 0.15 # 15%
```

Comment:
Let's make a note on what THRESHOLD does here and/or rename it to something a little more descriptive like ReceivePointChangeThreshold.